app.js

```diff
@@ -1,9 +1,9 @@
 const express = require('express')
 const app = express()
-const { getApi, getTopics, getArticleById } = require('./contollers/index')
+const { getApi, getTopics, getArticleById, getArticles } = require('./contollers/index')
 const { noEndpointError, requestErrors } = require('./serverErrors')
 
-app.use(express.json())
+app.use(express.json()) // This is here because I will forget it later and I'll spend hours stuck
```

Comment:
Not great 👀 But it's fine - as long as you are aware that you shouldn't have code in your codebase that you aren't using and it's bad practice I can't force you 😂